fairroot.sh

```diff
@@ -106,5 +113,6 @@ setenv CONFIG_DIR \$::env(VMCWORKDIR)/common/gconfig
 prepend-path PATH \$::env(FAIRROOT_ROOT)/bin
 prepend-path LD_LIBRARY_PATH \$::env(FAIRROOT_ROOT)/lib
 prepend-path ROOT_INCLUDE_PATH \$::env(FAIRROOT_ROOT)/include
+prepend-path ROOT_INCLUDE_PATH \$::env(FAIRROOT_ROOT)/include_tree
```

Comment:
Not needed if we do everything inside `include`

Reply:
I've thought of this, but I figured it makes sense to split the two, as `include_tree` is a total fake, just for cling.
Furthermore, if we created those dir symlinks inside `include` pointing to `..`, we'd get a cycle, which is not bad in itself but we de facto generate even more spurious paths.

Reply:
Loops aren't much of a problem as long as we use symlinks and not hard links (`find`, for instance, should deal with them neatly).

Reply:
Yes, but that's what I meant: loops are not bad in itself, but they mean that a downstream can do something like `#include <sink/sink/sink/sink/sink/sink/FairSomethingSomething.h>`.
Keeping a separate fake include dir which is only added to `ROOT_INCLUDE_PATH` and is not a standard include path picked up by `find_package` avoids this issue.